core/datainput.gms

```diff
@@ -733,7 +773,7 @@ pm_regiEarlyRetiRate(t,regi,"biohp")   = 0.5 * pm_regiEarlyRetiRate(t,regi,"bioh
 
 $ifthen.tech_earlyreti not "%c_tech_earlyreti_rate%" == "off"
 loop((ext_regi,te)$p_techEarlyRetiRate(ext_regi,te),
-  pm_regiEarlyRetiRate(t,regi,te)$(regi_group(ext_regi,regi) and (t.val lt 2035 or sameas(ext_regi,"GLO"))) = p_techEarlyRetiRate(ext_regi,te);
+  pm_regiEarlyRetiRate(t,regi,te)$(regi_group(ext_regi,regi) and (t.val lt 2065 or sameas(ext_regi,"GLO"))) = p_techEarlyRetiRate(ext_regi,te);
```

Comment:
Why is it that the early retirement zone expands?

Reply:
with "lt 2035", what happens is that if for some scenario, one would like to set for example no early retirement of pc for IND, one would think setting 'c_tech_earlyreti_rate = "..., IND_regi.pc 0"' will do the trick, but it actually only takes care of initial years up till 2030, but afterwards it will be 9% *per year* of coal that will be early retired (again, in REMIND coal is never profitable), which means in around 10 years, almost all coal can be retired. This makes simulating the political economy difficulty of early retiring coal very difficult, so one quick trick would be to extend it till middle of the century. 

in short, it allows regions to have longer periods to have designated early retirement rate (specified by c_tech_earlyreti_rate), but if one wants one can also make it a flag, @stephenbi @Renato-Rodrigues have an opinion?